portal/server/routes/static.js

```diff
@@ -8,7 +8,7 @@ router.get('/.well-known/security.txt', (req, res) => {
   res.write('Contact: https://www.gov.uk/contact/govuk\n');
   res.write('Contact: https://hackerone.com/7af14fd9-fe4e-4f39-bea1-8f8a364061b8/embedded_submissions/new\n');
   res.write('Contact: https://get-a-guarantee-for-export-finance.service.gov.uk/feedback\n');
-  res.write('Expires: 2024-02-07T00:00:00.000Z\n');
+  res.write('Expires: 2026-01-17T00:00:00.000Z\n');
```

Comment:
Thoughts on creating a const/function for this? E.g `twoYearsFromNow`? So we don't have to remember/forget to update it in 2026.